frigate/events/audio.py

```diff
@@ -168,9 +174,12 @@ def detect_audio(self, audio) -> None:
         if not self.feature_metrics[self.config.name]["audio_enabled"].value:
             return
 
-        waveform = (audio / AUDIO_MAX_BIT_RANGE).astype(np.float32)
+        audio_as_float = audio.astype(np.float32)
+        waveform = audio_as_float / AUDIO_MAX_BIT_RANGE
         model_detections = self.detector.detect(waveform)
 
+        self.calculate_audio_levels(audio_as_float)
```

Comment:
At what rate is this called? Are we constantly publishing messages for these values?

Reply:
that's a good point, the ffmpeg command outputs an audio frame once a second which is likely too frequent for this message to be published.